test/system/500-networking.bats

```diff
@@ -677,16 +677,20 @@ EOF
 @test "podman run port forward range" {
     for netmode in bridge slirp4netns:port_handler=slirp4netns slirp4netns:port_handler=rootlesskit; do
         local range=$(random_free_port_range 3)
-        local port="${test%-*}"
-        local end_port="${test#-*}"
```

Comment:
SUBTLE BUT IMPORTANT! Note the change of dash-asterisk to asterisk-star in this one!